chromium_src/ios/chrome/browser/application_context_impl.mm

```diff
@@ -311,6 +312,12 @@ void BindNetworkChangeManagerReceiver(
   return nullptr;
 }
 
+BreadcrumbPersistentStorageManager*
+ApplicationContextImpl::GetBreadcrumbPersistentStorageManager() {
+  DCHECK(thread_checker_.CalledOnValidThread());
+  return nullptr;
```

Comment:
cc @Brandon-T we'll need to test in the app to make sure this doesn't cause a crash

Reply:
It looks like it's called in 2 places:
1. https://source.chromium.org/chromium/chromium/src/+/master:ios/chrome/app/main_controller.mm;l=962 which does check for `nullptr`, and
2. https://source.chromium.org/chromium/chromium/src/+/master:ios/chrome/browser/metrics/mobile_session_shutdown_metrics_provider.mm;l=364, which doesn't, so this would crash.